### DISCUSSION TITLE
Possibility to change the browser used?

### DISCUSSION THREAD

**Discussion by zyberzebra** [aws-azure-login/aws-azure-login#334] [Q&A]
Would it be possible to not use chromium for mode-gui ?

**Comment by fok666** [comment]
Turns out you can. Just set `PUPPETEER_EXECUTABLE_PATH` environment variable to point to a browser binary:

Example:

`export PUPPETEER_EXECUTABLE_PATH='/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge'`
